microsoft/react-native-test-app
# fix: Add support for React Native 0.64

### Description

See title. This PR will not be merged until 0.64.0 is officially released.

### Blocking issues

- [x] APK is missing React Native `.so` files
- [x] `pod install --project-directory=ios` fails
- [ ] 0.64.0 is released

### Platforms affected


### Test plan

1. Run `yarn set-react-version 0.64`
2. Build and run iOS test app
    1. `pod install --project-directory=ios` outside `ios` folder should work
    2. `pod install` inside `ios` folder should work
3. Build and run Android test app
